### DISCUSSION TITLE
Default Costumes

### DISCUSSION THREAD

**Discussion by P-Star7** [Wohlstand/TheXTech#278] [Ideas]
Here is the area where we can discuss which costumes should be included defaultly as examples with TheXTech if costume support is added.

* denotes costume ideas that I still need to make by myself if people are interested in it.

Minimum pack (only two costumes per character, to allow selection of the same character in 2-player mode but with a distinct appearance)
Mario: SMB3 Wario*
Luigi: SMB3 Waluigi*
Peach: SMB2 Daisy
Toad: SMB2 Toadette*
Link: Zelda 2 Zelda* or Dark Link (in same SMB3 style as SMBX Link)

Expanded pack:

Mario:
Default
SMB3 Wario*
Demo
SMB1 Mario*
SMB2 Mario*
SMW Mario
SMW Luigi (SMW)
SMW Luigi (SMAS)
SMW Luigi (SMM2)
SMW Toad
SMW Toadette

Luigi
Default
SMB3 Waluigi*
Iris
SMB1 Luigi
SMB2 Luigi*
SMW Luigi (SNES)
SMW Luigi (SMAS)
SMW Luigi (SMM2)*

Peach
Default
Kood (making sure that he has correct forms 2-7 hitbox unlike ASMBXT)
SMB2 Daisy
SMB2 Rosalina
SMB1 Peach*
SMB3 Peach*
SMW Peach*

Toad
Default
SMBX 1.3 Toad (SMA to SNES conversion)
Blue Toad
Yellow Toad
SMB2 Toadette*
SMB1 Toad*
SMB3 Toad
SMW Toad
SMW Toadette*

Link
Default
Zelda 2 Zelda*
Zelda 2 Dark Link*
Sheath
ALTTP Link*
BS The Legend of Zelda Link*
Note: Latter two costumes will come when Wohlstand adds configuration of Link's sword offset/hitbox into calibrator.

**Comment by LoveBodhi** [comment]
Ideas for Flower Angel costume:

- Mario = Yukushiro Ai（小爱）
- Luigi = Phil（心岚）
- Peach = Berry（贝瑞）
- Toad = Savilly（萨维利）
- Link = Erik
